js-miniapp-sdk/README.md

```diff
@@ -81,21 +81,79 @@ miniApp.requestLocationPermission()
 	});
 ```
 
-### 4. Show Interstitial Ad
+### 4. Show Ads
 
-Mini App JS SDK makes it possible to display Interstitial ads during the execution of a mini-app. MiniApp#showInterstitialAd facilitates the display of the Ad while pairing with the Mini App Native SDKs. The results of the display action are captured as [InterstitialAdResponse](src/types/responseTypes/interstitial/index.ts).
-Note: This requires the mini-app to have obtained/enrolled the required ad units with the host app.
+Mini App SDK allows to display ads upon requesting from Mini apps requesting from Mini apps with ad unit id.
+This requires to pass in ad unit id to the host app in order to load the ad associated with the ad unit id.
+Currently two ad types are supported,
+1. Interstitial
+2. Rewarded
 
-There is an adType key in the response type which is a value from AdTypes enum and the value for this response is set to `AdTypes.INTERSTITIAL`
+**Interstitial**
+The results of the display `Interstitial` ad action is captured as [InterstitialAdResponse](src/types/responseTypes/interstitial/index.ts).
 
 ```javascript
-miniApp.showInterstitialAd()
+const adUnitID = 'xxx-xxx-xxxxxxxxxxxxx';
+miniApp.showInterstitialAd(adUnitID)
 	.then(response => {
 		console.log(response); // Successful response
 	}).catch(error => {
 		console.error(response); // Error
 	});
```

Comment:
I think we need to include the complete example of loading the Ad first and then displaying the Ad.

```suggestion
const adUnitID = 'xxx-xxx-xxxxxxxxxxxxx';

miniApp.loadInterstitialAd(adUnitID)
    .then(response => {
        miniApp.showInterstitialAd(adUnitID)
            .then(response => console.log(response) )
            .catch( error => console.error(response) );
    })
    .catch( error => console.error(response) );
```